### PR TITLE
:tada::one::zero: pos_journal_pin

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Comment by gustavovalverde** [comment]
When selecting the Journal, the popup says: **Select User**, it should be **Authorized Users**.

There's also a **bad side-effect**: If I do the authorization with the Manager PIN, it shouldn't automatically change to the manager. The PIN is just for validation, not for changing the user. <--- **This must be taken into consideration for pos_pin** as the user must not change to the manager.

File: pos_journal_pin/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+============================
+ Ask Manager to use journal
```
Comment (yelizariev) [inline pos_journal_pin/README.rst]: POS Payments by Manager's PIN

File: pos_journal_pin/README.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+============================
+
+Ask Manager to use journal
```
Comment (yelizariev) [inline pos_journal_pin/README.rst]: A little bit more details would be greate

File: pos_journal_pin/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+============================
+ Ask Manager to use journal
+============================
+
+Installation
+============
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
```
Comment (yelizariev) [inline pos_journal_pin/doc/index.rst]: No Usage instruction

File: pos_journal_pin/models/models.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+# Copyright 2018 Kolushov Alexandr <https://it-projects.info/team/KolushovAlexandr>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.journal'
+
+    ask_manager = fields.Boolean("Manager's permission")
```
Comment (yelizariev) [inline pos_journal_pin/models/models.py]: It's not clear what is for

File: pos_pin/README.rst [hunk 5]
```diff
@@ -1,29 +1,50 @@
...
+It adds ``sudo_custom`` method to be used in other modules. Unlike the build-in ``sudo`` function, the module allows
 specify access group that user must have to confirm an action.
 
 Credits
 =======
 
 Contributors
 ------------
-* krotov@it-projects.info
+* `Krotov Stanislav <https://github.com/ufaks>`__
```
Comment (yelizariev) [inline pos_pin/README.rst]: https://www.it-projects.info/team/ufaks

Comment (KolushovAlexandr) [reply pos_pin/README.rst]: 404: Page not found!

**Comment by ilmir-k** [comment]
> We need html description to publish it.

I'll prepare it

**Comment by KolushovAlexandr** [comment]
@yelizariev 'You can send pos_pin only in a separate PR'
the commit with pos_pin is already merged into the main branches now
I've added it here to show the resulting functionality, that entering a manager PIN doesn't change the cashier. I can delete it now or before merging the PR. 
And about your comment about pos_pin I'll make the change in the different PR

File: pos_pin/README.rst (hunk 5 above)
Comment (yelizariev) [reply pos_pin/README.rst]: I know. The idea is to fix 404

**Pull request merged**